modules/catalog/src/testFixtures/java/org/apache/ignite/internal/catalog/CatalogTestUtils.java

```diff
@@ -155,10 +161,10 @@ public static CatalogManager createCatalogManagerWithTestUpdateLog(String nodeNa
 
         return new CatalogManagerImpl(new TestUpdateLog(clock), clockWaiter) {
             @Override
-            public void start() {
-                clockWaiter.start();
+            public CompletableFuture<Void> start() {
+                CompletableFuture<Void> fut = clockWaiter.start();
 
-                super.start();
+                return allOf(new CompletableFuture[]{fut, super.start()});
```

Comment:
no need to create an array explicitly here, `allOf` has a vararg argument

Reply:
done